fix(nuxt): enable router when `app/router.options.ts` file is present (#8193)
nuxt/framework: docs/content/2.guide/2.directory-structure/1.pages.md

Nuxt provides a file-based routing to create routes within your web application using [Vue Router](https://router.vuejs.org) under the hood.

::alert{type="info"}

> This directory is **optional**, meaning that [`vue-router`](https://router.vuejs.org) won't be included if you only use [app.vue](/guide/directory-structure/app) (unless you provide an [`router.options.ts`](/guide/directory-structure/pages#router-options), reducing your application's bundle size.

Reviewer: ```suggestion
This directory is **optional**, meaning that [`vue-router`](https://router.vuejs.org) won't be included if you only use [app.vue](/guide/directory-structure/app) (unless you set `pages: true` in `nuxt.config` or have a [`app/router.options.ts`](/guide/directory-structure/pages#router-options)), reducing your application's bundle size.
```
